region/maridia/inner-green.json

```diff
@@ -635,6 +635,27 @@
               ],
               "openEnd": 1
             }
+          ],
+          "leaveWithGModeSetup": [
+            {
+              "strats": [
+                {
+                  "name": "Get Hit By Evir Projectile",
+                  "notable": false,
+                  "requires": [ "Gravity" ]
+                },
+                {
+                  "name": "Get Hit By Evir Projectile Suitless",
+                  "notable": false,
+                  "requires": [
+                    "canSuitlessMaridia",
+                    "canTurnaroundSpinJump",
+                    "canTrickyJump"
+                  ],
+                  "note": "Start with Samus' toe barely on the last tile of the platform. Move left then do a short spin jump back to the door and land before the projectile gets to Samus."
```

Comment:
I prefer small jumping to the left tile then small jumping back to the door.  Breaking spin to land faster both times.  Its still really hard to land right next to the transition.